### ISSUE TITLE
Set mem_len and lmem_len =0, the result get worse

### ISSUE THREAD

**Issue by wgl1999** [lucidrains/memory-transformer-xl#1]
Hello lucidrains,

I truly appreciate your work! I've been training it on my own data to classify intention. 

However, when I set mem_len to 0, I noticed that the results worsened. I used the same hyperparameters for this experiment. Could using memory make optimizing the task more difficult? Should I consider setting a lower learning rate or something else?

**Comment by wgl1999** [comment]
low lr works!
